tests/FixturesLoader.php

```diff
@@ -102,6 +102,10 @@ protected function loadImportFiles(array $files)
         //         "/project/features/Context/catalog/footwear/jobs.yml"
         //         "/project/features/PimEnterprise/Behat/Context/../../../Context/catalog/footwear/jobs.yml"
         // ]
+        if (empty($files)) {
```

Comment:
Maybe that's not the right optimization. But if in line 54 filesByType['import'] is empty (like in the technical_sql catalog), I don't see why we should load anything in the first place.
I just wanted to raise awareness about this method